src/number/n_Zn.jl

```diff
@@ -89,6 +89,14 @@ function show(io::IO, c::N_ZnRing)
    print(io, "Residue Ring of Integer Ring modulo ", characteristic(c))
 end
 
+function AbstractAlgebra.expressify(n::n_Zn; context = nothing)::Any
+  x = BigInt()
+  GC.@preserve n begin
+    ccall((:__gmpz_set, :libgmp), Cvoid, (Ref{BigInt},  Ptr{Any}), x, n.ptr.cpp_object)
```

Comment:
This code is also not tested by the test suite, would be good if it was to check the `ccall`.

Also, I think it would be quite useful and natural to have a function that allows converting an `n_Zn` to `BigInt` (or `fmpz` or `n_Z`?). Perhaps we should copy/move the above code into a `BigInt` constructor that one can use to do so?

Reply:
I would rather not include the discussion on what `BigInt(::n_Zn)` returns in this PR here (lift to `[0, n)` or `[-n/2,n/2]`).